src/eth1spec/main.py

```diff
@@ -1,15 +1,20 @@
+# flake8: noqa
+
 import argparse
 import json
 
 from . import spec, trie
+from .eth_types import Address, Uint
 
 # sys.setrecursionlimit(10000)
 
 
-def main():
+def main() -> None:
     parser = argparse.ArgumentParser(description="Process some integers.")
 
-    parser.add_argument("--input.alloc", dest="alloc", action="store")
+    parser.add_argument(
+        "--input.allocation", dest="allocation", action="store"
```

Comment:
```suggestion
        "--input.alloc", dest="allocation", action="store"
```
This should be `alloc` to match the `t8n` [spec](https://github.com/ethereum/go-ethereum/tree/master/cmd/evm#basic-usage).